x-pack/legacy/plugins/apm/public/components/app/Main/route_config/index.tsx

```diff
@@ -149,3 +160,26 @@ export const routes: BreadcrumbRoute[] = [
     name: RouteName.TRANSACTION_NAME
   }
 ];
+
+if (npStart.core.injectedMetadata.getInjectedVar('apmServiceMapEnabled')) {
+  routes.push(
+    {
+      exact: true,
+      path: '/service-map',
+      component: () => <Home tab="service-map" />,
+      breadcrumb: i18n.translate('xpack.apm.breadcrumb.serviceMapTitle', {
+        defaultMessage: 'Service Map'
+      }),
+      name: RouteName.SERVICE_MAP
+    },
+    {
+      exact: true,
+      path: '/services/:serviceName/service-map',
+      component: () => <ServiceDetails tab="service-map" />,
+      breadcrumb: i18n.translate('xpack.apm.breadcrumb.serviceMapTitle', {
+        defaultMessage: 'Service Map'
+      }),
+      name: RouteName.SINGLE_SERVICE_MAP
+    }
+  );
+}
```

Comment:
Any reason `'/services/:serviceName/service-map'` is added unconditionally but these are not?

Reply:
That was actually a duplicate from resolving a merge conflict. Fixed in 799b5e9b9f9.